tutorials/workshop_examples/launch/turtlebot_slam.launch

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<launch>
+
+<arg name="record" default="False" doc="Record results in a bag file"/>
+<arg name="play" default="False" doc="Play recorded bag file"/>
+<arg name="bagfile" default="" doc="Play recorded bag file"/>
+
+<node name="rviz" pkg="rviz" type="rviz" args="-d $(find turtlebot3_slam)/rviz/turtlebot3_slam.rviz" />
+
+<group if="$(arg play)">
+  <node pkg="rosbag" type="play" name="rosbag_play" args="$(env HOME)/ROS_workshop/data/$(arg bagfile) /cmd_vel:=/cmd_vel_dev_null"/>
+  <arg name="use_sim_time" value="true"/>
```

Comment:
Should not `use_sim_time` be inside `param` tags ? `use_sim_time` is set on the parameter server.

Reply:
Yes you are right, this might explain some problems i've been having